src/metabase/api/field.clj

```diff
@@ -396,53 +396,52 @@
     (api/check-403 (mi/can-read? search-field))
     (search-values field search-field value mw.offset-paging/*limit*)))
 
-(defn remapped-value
-  "Search for one specific remapping where the value of `field` exactly matches `value`. Returns a pair like
+(defn remapped-values
+  "Search for remapped values where the value of `field` exactly matches one of `values`. Returns a list of pairs like
 
       [<value-of-field> <value-of-remapped-field>]
 
    if a match is found.
 
    For example, with the Sample Database, you could find the name of the Person with ID 20 as follows:
 
-      (remapped-value <PEOPLE.ID Field> <PEOPLE.NAME Field> 20)
-      ;; -> [20 \"Peter Watsica\"]"
-  [field remapped-field value]
+      (remapped-values <PEOPLE.ID Field> <PEOPLE.NAME Field> [20])
+      ;; -> [[20 \"Peter Watsica\"]]"
+  [field remapped-field values]
   (try
     (let [field   (follow-fks field)
           results (qp/process-query
                    {:database (db-id field)
                     :type     :query
                     :query    {:source-table (table-id field)
-                               :filter       [:= [:field (u/the-id field) nil] value]
+                               :filter       (concat [:= [:field (u/the-id field) nil]] values)
```

Comment:
It might a good idea keep filter as a vector.
```suggestion
                               :filter       (into [[:= [:field (u/the-id field) nil]]] values)
```